### PR TITLE
API to force refresh of thread TLH

### PR DESCRIPTION
**Pull request by theresa-m**
Signed-off-by: Theresa Mammarella <tmammare@redhat.com>

### CODE REVIEW

**Comment by dmitripivkine** [comment]
@amicic FYI

File: gc/base/TLHAllocationInterface.cpp [hunk 1]
```diff
@@ -161,6 +161,22 @@ MM_TLHAllocationInterface::allocateFromTLH(MM_EnvironmentBase *env, MM_AllocateD
...
+	if (allocDescription->getNonZeroTLHFlag()) {
+		result = _tlhAllocationSupportNonZero.refresh(env, allocDescription, false, true);
+	} else
+#endif /* defined(OMR_GC_NON_ZERO_TLH) */ 
+	{
+		result = _tlhAllocationSupport.refresh(env, allocDescription, false, true);
+	}
+
+	return result;
+}
```
Comment (dmitripivkine) [inline gc/base/TLHAllocationInterface.cpp:178]: If dual TLH mode is used this function is going to refresh one or another. So if intention is to refresh both TLHs either you should manage to do it here or alternatively call this function twice for each type of TLH

**Comment by 0xdaryl** [comment]
Just wondering what the next step is for this PR and whether it is still active.

**Comment by dmitripivkine** [comment]
I think this change is not required.
@theresa-m @jonoommen Is it correct?

**Comment by jonoommen** [comment]
> I think this change is not required.
> @theresa-m @jonoommen Is it correct?

I am not sure, @theresa-m would know better regarding whether or not this change is still required.

**Comment by theresa-m** [comment]
> I think this change is not required.
> @theresa-m @jonoommen Is it correct?

This should not be necessary anymore.

**Pull request closed without merging**